larpandoracontent/LArVertex/CandidateVertexCreationAlgorithm.cc

```diff
@@ -34,7 +34,10 @@ CandidateVertexCreationAlgorithm::CandidateVertexCreationAlgorithm() :
     m_extrapolationNSteps(200),
     m_extrapolationStepSize(0.1f),
     m_maxCrossingSeparationSquared(2.f * 2.f),
-    m_minNearbyCrossingDistanceSquared(0.5f * 0.5f)
+    m_minNearbyCrossingDistanceSquared(0.5f * 0.5f),
+    m_reducedCandidates(true),
+    m_par1(2),
```

Comment:
Should try to make these more self-describing. E.g. even CandidateReductionSchemePar1 would be a bit clearer.